packages/flutter_tools/lib/src/android/gradle_utils.dart

```diff
@@ -83,9 +104,8 @@ class GradleUtils {
       return gradle.absolute.path;
     }
     throwToolExit(
-      'Unable to locate gradlew script. Please check that ${gradle.path} '
-      'exists or that ${gradle.dirname} can be read.'
-    );
+       'Unable to locate gradlew script. Please check that ${gradle.path} '
```

Comment:
`gradlew` is actually `gradlew.bat` on Windows, afaik.

Maybe "Unable to locate the Gradle Wrapper script" or something like this?